asapdiscovery-ml/pyproject.toml

```diff
@@ -19,13 +19,6 @@ classifiers = [
     "Operating System :: MacOS",
 ]
 dependencies = [
-    "torch",
-    "pandas",
-    "upsetplot",
-    "seaborn",
-    "dgllife",
-    "torch-cluster",
-    "e3nn"
```

Comment:
do we no longer need these because they're already covered by `mtenn` ?

Reply:
Yep :) they are picked up in the environment file.